charts/nginx-ingress/templates/controller-role.yaml

```diff
@@ -42,8 +42,6 @@ rules:
   - coordination.k8s.io
   resources:
   - leases
-  resourceNames:
-  - {{ .Values.controller.reportIngressStatus.leaderElectionLockName }}
```

Comment:
I'm curious, why is this removed?  Why not use `{{ include "nginx-ingress.leaderElectionName" . }}`

Reply:
@jjngx I guess we can keep it here, only suggested this to keep it in line with configmap leader election. (IC wonlt be able to list or get any other lease object) 
also makes sense why why verb: create is in separate definition as resource can't be referred there with name https://kubernetes.io/docs/reference/access-authn-authz/rbac/#referring-to-resources

Reply:
Removing it may be the correct approach.  I'm just interested pros/cons of having it included

Reply:
Does including it lead to a better least privilege model

Reply:
@pdabelf5 yes from the docs it seems like it so unless we'll have another lease object to watch, keeping should be okay

Reply:
thanks @pdabelf5 , @vepatel, addressed